adserver/validators.py

```diff
@@ -7,6 +7,46 @@
 from .utils import COUNTRY_DICT
 
 
+@deconstructible
+class TopicPricingValidator(BaseValidator):
+
+    """Field validator for per topic pricing for regions."""
+
+    message = _("Enter a valid value")
+    code = "topic-pricing-validator"
+    limit_value = None  # required for classes extending BaseValidator
+
+    def __init__(self, message=None):
+        """Initialization for the topic pricing validator."""
+        if message:
+            self.message = message
+
+    def __call__(self, value):
+        from .models import Topic  # noqa
```

Comment:
Guessing there's a reason this is a nested import, but probably good to note it here?

Reply:
The validators are used to validate the models. It would be circular without this. I'll note it.